sdks/go/test/integration/wordcount/wordcount.go

```diff
@@ -18,15 +18,13 @@ package wordcount
 
 import (
 	"context"
-	"regexp"
```

Comment:
We can remove this change. At best, we mostly want to remove the blank line between "strings" and "fmt".